_pages/en/strategy-technology-management.md

```diff
@@ -0,0 +1,338 @@
+
+This goal directly points to the efforts by ESDC's Chief Data Officer and Enterprise Information Management team.
+
+Progress on this goal is measured by:
+
+A\) _Percentage of Programs that identified technology's on-going expenses in their periodic Treasury Board Submission Renewals_ (data source: Programs Treasury Board Submissions)
+
+B\) _Degree of Data and Information literacy_ (data source: Data and IM Strategies desired outcomes and measurement methods)
+
+***3\) Modernize IITB's software delivery capabilities***
+
+This goal directly points to Foundational Capabilities identified in the [IITB Strategic
+Priorities](https://014gc.sharepoint.com/sites/IITB-DGIIT-Gov-New-Nouveau/SitePages/IITB's-Strategic-Priorities-One-Pager.aspx).
+In particular "higher velocity and quality through the adoption of modern ways of working" and "maturing cloud services capabilities".
+
+Technology is no longer a "package" that you purchase at a store and install.
+Technology, including infrastructure, has become software services that are configured, consumed, and that keeps on improving over time.
+Demands for their use require a variety of internal IT services for their effective and secure delivery, but ultimately when a Program seeks technology services, they seek to obtain software services.
+
+Project demands, maintenance, migration to the public cloud or end state data centre, are all sources of requirements that inform the "Application's" roadmap (a grouping of software).
+The ability for an IT Organization to execute and make progress on all demands submitted to it is heavily dependent on its ability to perform more frequent and smaller software changes.
+
+Progress on this goal is measured by[^8]:
+
+A\) _Decrease in lead time for change_: the time code changes take to go from check-in to release in production (data source: TBD, may be a combination of Application's source code repository and IT change record)[^9]
+
+B\) _Increase in deployment frequency_: the rate at which software is deployed to production or released to end users (data source: IT change record)
+
+C\) _Decrease in change failure rates_: the change failure rate measured by how often deployment failures occur in production that require immediate remedy (data source: IT change record)
+
+D\) _Decrease in time to restore_: the time it takes from detecting a user impacting incident to remediating it (data source: National Service Desk record)
+
+E\) _Decrease in Cyber Security Vulnerabilities_: the number of cyber-security vulnerabilities registered per application (data source: Vulnerabilities Registries)
+
+## Coherent set of actions
+
+A coherent set of actions is necessary to align resources, internal policies, and maneuvers in a coordinated fashion to support each other as opposed to fighting against each other.
+This set of actions needs sufficient buy-in at a sufficient level of authority to provide cover for the personnel that will be doing the work.

```

Comment:
I would remove 'sufficient" from before "buy-in". I think if you have buy in it is automatically sufficient. :)